drivers/gles2/shaders/scene.glsl

```diff
@@ -930,11 +930,11 @@ LIGHT_SHADER_CODE
 		float G = G_GGX_2cos(cNdotL, alpha) * G_GGX_2cos(cNdotV, alpha);
 #endif
 		// F
-		//float F0 = 1.0;
-		//float cLdotH5 = SchlickFresnel(cLdotH);
-		//float F = mix(cLdotH5, 1.0, F0);
+		float F0 = 1.0;
+		float cLdotH5 = SchlickFresnel(cLdotH);
+		float F = mix(cLdotH5, 1.0, F0);
```

Comment:
Currently it's doing nothing. F0 is always 1.0 and so the mix() call is also 1.0. I need to check back how it was supposed to work, but was planned to do later.

Reply:
Oh, right, I remember it now. This is why I put a FIXME on that line a while ago actually (which got removed in 65fd37c).

Nevertheless, as I also noted in this PR, the value of F0 should be derived, from the IOR. Did Godot have a IOR texture/parameter?

Reply:
Or F0 itself can be a material parameter actually (scalar for dielectrics, color value for metals).
Otherwise, can just set it to a typical value of 0.04.

Reply:
I am not totally convinced of exposing it, as its not that useful for artists (who just dump stuff from Substance). Most engines I saw simply leave it at 0.04, but in the case of Godot given we use forward rendering, I can imagine exposing it may not hurt anyway,

Reply:
Ah, right Godot already has "specular" parameter for that.

@fracteed As you can see from the Filament docs (Table 4), F0=0.04 isn't a good value for skin, water/ice and various crystals, so I think it's important to have a knob for that.

@reduz, is it OK if I rename it to specular_reflectance, or just reflectance? It's pretty confusing right now (in the editor, "specular" is confusing because "specular" typically refers to specular texture of the so-called specular workflow, and in the shader code, specular is also sometimes used for specular BRDF).

Then we can do:

`vec3 F0 = mix(0.16 * reflectance * reflectance, albedo, metallic).`

Reply:
> Maybe there could even be a dropdown selection in the specular field for the various ior's as per this table. This is common in some commercial renderers so that the user doesn't have to guess or memorise these values?

Or this can simply be added to the docs .